### ISSUE TITLE
oger battle texture

### ISSUE THREAD

**Issue by Nes-player4Life** [gonetz/GLideN64#2711]
ogre battle seems to dump textures however once a user goes to open them in an editor such as photoshop the image is a transparent file.

**Comment by Xii-Nyth** [comment]
this also happens with a few textures in pd, most of the time its just the 40x30 screenshot of the background when entering the pause menu. but theres a few real ones which just seem to be gradients/shadows

**Comment by Xii-Nyth** [comment]
good point, might just be alpha set at 0, can try setting to max with levels in gimp easily. since they might not check github all the time Ill let them know on discord.
